doc/sphinx/source/projects/ad7191_iio/ad7191_iio.rst

```diff
@@ -0,0 +1,102 @@
+AD7191 IIO Application
+""""""""""""""""""""""
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `AD7191 <https://www.analog.com/en/products/ad7191.html>`_
+
+**Supported Evaluation Boards:**
+
+* `EVAL-AD7191 <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7191asdz.html>`_
+
+**Supported Carrier Boards:**
+
+* `SDP-K1  <https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/sdp-k1.html#eb-overview>`_
+
+============
+Introduction
+============
+
+This page gives an overview of using the ARM platforms supported 
+firmware example with Analog Devices AD7191 Evaluation board and SDP-K1 controller board. 
+This example code leverages the ADI developed IIO (Industrial Input Output) ecosystem to 
+evaluate the AD7191 device by providing a device debug and data capture support.
```

Comment:
nit: Can continue the default statement here: `The code provides support for STM32 platform using the SDP-K1 controller board.`